deployment/clouddeploy/gke-workers/environments/oss-vdb-test/debian-copyright-mirror.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: debian-copyright-mirror
+spec:
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: debian-copyright-mirror
+            env:
+            - name: GCS_PATH
+              value: gs://osv-test-cve-osv-conversion/debian_copyright
+            - name: WORK_DIR
+              value: /scratch
```

Comment:
nit: this `WORK_DIR` env can be moved to the base yaml, since it's the same in both environments.

Reply:
Thanks, fixed in f47b8a5

Out of interest, with all this inheritance, there's a non-trivial cognitive burden when looking at the files to determine what the resultant spec actually looks like. Is there something you can run to see the merged YAML?

Reply:
```
kubectl kustomize environments/oss-vdb-test > out.yaml
```
should output the kustomize'd yaml for that environment.